### ISSUE TITLE
Doesn't work with Meteor 1.3 (beta-11)

### ISSUE THREAD

**Issue by clayne11** [zodern/meteor-up#66]
I'm getting the following error when I try to deploy:

```
npm WARN deprecated such as scoped packages, offered by the primary npm
npm WARN deprecated registry. Consider upgrading to at least npm@2, if not the
npm WARN deprecated latest stable version. To upgrade to npm@2, run:
npm WARN deprecated
npm WARN deprecated   npm -g install npm@latest-2
npm WARN deprecated
npm WARN deprecated To upgrade to the latest stable version, run:
npm WARN deprecated
npm WARN deprecated   npm -g install npm@latest
npm WARN deprecated
npm WARN deprecated (Depending on how Node.js was installed on your system, you
npm WARN deprecated may need to prefix the preceding commands with `sudo`, or if
npm WARN deprecated on Windows, run them from an Administrator prompt.)
npm WARN deprecated
npm WARN deprecated If you're running the version of npm bundled with
npm WARN deprecated Node.js 0.10 LTS, be aware that the next version of 0.10 LTS
npm WARN deprecated will be bundled with a version of npm@2, which has some small
npm WARN deprecated backwards-incompatible changes made to `npm run-script` and
npm WARN deprecated semver behavior.
npm WARN package.json meteor-dev-bundle@0.0.0 No description
npm WARN package.json meteor-dev-bundle@0.0.0 No repository field.
npm WARN package.json meteor-dev-bundle@0.0.0 No README data
=> Starting meteor app on port:80
```

And then Meteor is never started.

A similar issue can be seen here: https://github.com/arunoda/meteor-up/issues/841

**Comment by madushan1000** [comment]
If there are no further logs, The app probably deploys correctly. Just the verification script can't verify the app is running. Try increasing the `deployCheckWaitTime` in the `mup.js`. Or this can be your firewall blocking requests on port 80. Check that too.

**Comment by clayne11** [comment]
This was fixed in a later release of Meteor 1.3

**Comment by skynightz93** [comment]
Clayne11, i try with rc4 and still this think dont fix. What version that you use?
